include/vicon/udp.h

```diff
@@ -40,9 +40,10 @@ class ObjectData
     using millimeter_t = units::length::millimeter_t;
 
 public:
-    ObjectData(const std::string &objectName)
-      : m_Name(objectName)
-    {}
+    ObjectData(const char *objectName)
```

Comment:
As we are using fixed size arrays you can use this slightly aggregious syntax to make that explicit:

```c++
ObjectData(char (&objectName)[24])
{
```

Reply:
Good point.